packages/flutter_tools/test/general.shard/ios/ios_project_migration_test.dart

```diff
@@ -1024,4 +1038,4 @@ class FakeIOSMigrator extends ProjectMigrator {
   String migrateLine(String line) {
     return line;
   }
-}
+}
```

Comment:
```
   info • Put a single newline at end of file • packages/flutter_tools/test/general.shard/ios/ios_project_migration_test.dart:1041:2 • eol_at_end_of_file

```

Reply:
Fixed in https://github.com/flutter/flutter/pull/114099/commits/2637c23e341c698f44dfa6c79b78152b74ac33ea.